### PR TITLE
Add missing hotkeys to User Guide

### PR DESCRIPTION
**Pull request by NotHyper-474**
### Linked Issues
Closes https://github.com/FunkinCrew/Funkin/issues/3697

Adds the Focus Camera and note context/options menu hotkeys that are currently not documented anywhere (except for the camera events in the changelog for v0.3.2).
If you have a better and/or shorter description to the hotkeys you're very welcome to suggest changes.

### CODE REVIEW

File: preload/data/ui/chart-editor/dialogs/user-guide.xml [hunk 1]
```diff
@@ -18,6 +18,8 @@
 					<item action="Select Note/Event" keybind="Left Click on Note" />
 					<item action="Select Multiple Notes/Events" keybind="Ctrl + Left Click" />
 					<item action="Remove Note/Event" keybind="Right Click on Note" />
```
Comment (Hundrec) [inline preload/data/ui/chart-editor/dialogs/user-guide.xml:20]:
```suggestion
					<item action="Remove Note/Event" keybind="Right Click on Note" />
					<item action="Show Note Options" keybind="Shift + Right Click on Note" />
```

**Review by Hundrec — CHANGES_REQUESTED** [review]
Just realized the shortcut mentioned in https://github.com/FunkinCrew/Funkin/issues/3338 isn't listed in this menu.

File: preload/data/ui/chart-editor/dialogs/user-guide.xml (hunk 1 above)
Comment (NotHyper-474) [reply preload/data/ui/chart-editor/dialogs/user-guide.xml:20]: Oh right, never noticed this shortcut wasn't mentioned. Thanks!

**Review by Hundrec — APPROVED** [review]

**Comment by NotHyper-474** [comment]
Wahoo :tada:

**Pull request merged**